### PR TITLE
Add logs about errors during adding peer

### PR DESCRIPTION
**Pull request by Rados13**
## Acknowledging the stipulations set forth:
 - [x] I hereby confirm that a Pull Request involving updates to the Software Development Kit (SDK) has been smoothly merged, currently awaits processing, or is otherwise deemed unnecessary in this context.
 - [x] I also affirm that another Pull Request, specifically addressing updates to the documentation body (commonly referred to as 'docs'), has either been successfully incorporated, is in the process of review, or is considered superfluous under the prevailing circumstances.

### CODE REVIEW

File: lib/fishjam/room_service.ex [hunk 1]
```diff
@@ -133,6 +133,8 @@ defmodule Fishjam.RoomService do
 
   @impl true
   def handle_call({:create_room, config}, _from, state) do
+    Logger.info("Start create room")
+
```
Comment (sgfn) [inline lib/fishjam/room_service.ex:137]: Shouldn't this be `debug` as well?

File: lib/fishjam/room_service.ex [hunk 2]
```diff
@@ -148,7 +150,13 @@ defmodule Fishjam.RoomService do
       {:reply, {:ok, room, Fishjam.address()}, state}
     else
       {:error, :room_already_exists} = error ->
+        Logger.warning("Room creation failed, because it already exists")
+
         {:reply, error, state}
+
+      reason ->
+        Logger.warning("Room creation failed with reason: #{inspect(reason)}")
+        {:reply, :room_doesnt_start, state}
```
Comment (sgfn) [inline lib/fishjam/room_service.ex:159]: 1. Shouldn't this be `{:error, :room_doesnt_start}`?
2. This needs to be handled in `RoomController` as well

File: lib/fishjam_web/controllers/peer_controller.ex [hunk 3]
```diff
@@ -86,19 +89,30 @@ defmodule FishjamWeb.PeerController do
       |> render("show.json", assigns)
     else
       :error ->
-        {:error, :bad_request, "Invalid request body structure"}
+        msg = "Invalid request body structure"
+        Logger.warning(msg)
```
Comment (sgfn) [inline lib/fishjam_web/controllers/peer_controller.ex:93]: Shouldn't we specify the action that failed? I.e. `"Unable to add peer to room #{room_id}: " <> msg`?

File: lib/fishjam_web/controllers/room_controller.ex [hunk 4]
```diff
@@ -107,6 +107,10 @@ defmodule FishjamWeb.RoomController do
         room_id = Map.get(params, "roomId")
         {:error, :bad_request, "Cannot add room with id \"#{room_id}\" - room already exists"}
 
+      {:error, :room_doesnt_start} ->
+        room_id = Map.get(params, "roomId")
+        {:error, :bad_request, "Cannot add room with id \"#{room_id}\" - adding failed"}
```
Comment (sgfn) [inline lib/fishjam_web/controllers/room_controller.ex:112]: How about "unexpected error" instead? I feel like we're repeating ourselves with "adding failed"

**Review by sgfn — APPROVED** [review]

File: lib/fishjam_web/controllers/room_controller.ex [hunk 5]
```diff
@@ -74,6 +75,8 @@ defmodule FishjamWeb.RoomController do
   end
 
   def create(conn, params) do
+    Logger.debug("Start creating room")
```
[the remark this answers is not in the document]
Comment (Rados13) [reply lib/fishjam_web/controllers/room_controller.ex:78]: With creating room, is that difference that the log from `room_controller` will log on a node to which the request comes, and the log from `room_service` will log on the node where this room is created? That's why I see benefits in having these two logs, but maybe the messages should be different.

**Pull request merged**